website/docs/api/large-language-models.mdx

```diff
@@ -302,6 +303,171 @@ max_n_words = 20
 path = "summarization_examples.yml"
 ```
 
+### EL (Entity Linking) {id="nel"}
+
+The EL links recognized entities (see [NER](#ner)) to those in a knowledge base
+(KB). The EL task prompts the LLM to select the most likely candidates from the
+KB, whose structure can be arbitrary.
+
+Note that the documents processed by the entity linking task are expected to
+have recognized entities in their `.ents` attribute. This can be achieved by
+either running the [NER task](#ner), using a trained spaCy NER model or setting
+the entities manually prior to running the EL task.
+
+In order to be able to pull data from the KB, an object implementing the
+`CandidateSelector` protocol has to be provided. This requires two functions:
+(1) `__call__()` to fetch candidate entities for entity mentions in the text
+(assumed to be available in `Doc.ents`) and (2) `get_entity_description()` to
+fetch descriptions for any given entity ID. Descriptions can be empty, but
+ideally provide more context for entities stored in the KB.
+
+`spacy-llm` provides a `CandidateSelector` implementation
+(`spacy.CandidateSelector.v1`) that leverages a a spaCy knowledge base -as used
+in an `entity_linking` component - to select candidates. This knowledge base can
```

Comment:
```suggestion
in an `entity_linking` component- to select candidates. This knowledge base can
```

Reply:
Fixed in 99f23e87bd3878b303d4a8b62023bcd7899950b3.